paladin-quests.lic

```diff
@@ -92,72 +91,73 @@ class PaladinQuests
     when 'the Citizenship Office'
-      echo '***YOU DO NOT MEET THE REQUIREMENTS***'
-      echo "***PLEASE SUPPLY THIS LOG AS AN ISSUE IF YOU GET A 'NO MATCH FOUND' MESSAGE***"
+      message '***YOU DO NOT MEET THE REQUIREMENTS***'
+      message "***PLEASE SUPPLY THIS LOG AS AN ISSUE IF YOU GET A 'NO MATCH FOUND' MESSAGE***"
       exit
     end
 
-    echo '***PAUSING SCRIPT. FIND THE ROOM DISPLAYED IN THE VISION. IT WILL BE IN THIS TOWN!***'
-    echo '***UNPAUSE WHEN READY ( ;unpause paladin-quests )***'
+    message '***PAUSING SCRIPT. FIND THE ROOM DISPLAYED IN THE VISION. IT WILL BE IN THIS TOWN!***'
+    message '***UNPAUSE WHEN READY ( ;unpause paladin-quests )***'
     pause_script('paladin-quests')
 
-    echo '***OK YOU FOUND THE ROOM***'
-    while result != ~ /In the fountain lies/i
+    message '***OK YOU FOUND THE ROOM***'
+    result = bput('meditate', /In the fountain lies/i)
+    while result !~ /In the fountain lies/i
       result = bput('meditate', /In the fountain lies/i)
       unless result =~ /In the fountain lies/i
-        echo '***LOOKS LIKE THE WRONG ROOM. TRY AGAIN***'
+        message '***LOOKS LIKE THE WRONG ROOM. TRY AGAIN***'
         pause_script('paladin-quests')
       end
     end
 
     case bput('look in fountain', 'As you gaze', 'I could not find', 'What were you')
     when 'I could not find'
-      echo '***SOMETHING WENT REALLY WRONG. TRY AGAIN***"'
+      message '***SOMETHING WENT REALLY WRONG. TRY AGAIN***"'
       exit
     end
 
-    case bput('look painting in fountain', 'a Paladin fallen in battle')
-    when result =~ /a Paladin fallen in battle/i
-      result = bput('get painting in fountain', 'chosen wisely')
-      if result =~ /chosen wisely/i
-        echo '***CONGRATS YOU NOW KNOW THE GLYPH OF BONDING***'
-        exit
-      end
-    end
-
-    echo '***YOU ARE TAKING CHARGE AGAIN!. You need to look at each item in the fountain and choose wisely between them.' \
+    message '***YOU ARE TAKING CHARGE AGAIN!. You need to look at each item in the fountain and choose wisely between them.' \
          ' Clue: Paladin defend those that are unable to defend themselves***'
     waitfor('chosen wisely')
-    echo '***PLEASE SUBMIT AN ISSUE ON GITHUB WITH YOUR FOUNTAIN ITEMS***'
-    echo '***CONGRATS YOU NOW KNOW THE GLYPH OF BONDING***'
+    message '***PLEASE SUBMIT AN ISSUE ON GITHUB WITH YOUR FOUNTAIN ITEMS***'
+    message '***CONGRATS YOU NOW KNOW THE GLYPH OF BONDING***'
   end
 
   def light
-    echo '***UNSUPPORTED AT THIS TIME. PLEASE CONSIDER MAKING AN ISSUE AND' \
+    message '***UNSUPPORTED AT THIS TIME. PLEASE CONSIDER MAKING AN ISSUE AND' \
          ' PROVIDING THE LOGS FOR ADDITION***'
   end
 
   def mana
-    echo '***UNSUPPORTED AT THIS TIME. PLEASE CONSIDER MAKING AN ISSUE AND' \
-         ' PROVIDING THE LOGS FOR ADDITION***'
+    message '***PAUSING SCRIPT. MAKE SURE TO GET THE QUEST FROM THE CROSSING PALADIN GUILD LEADER (ask veri about glyph)!***'
```

Comment:
It would be nice to automate this part too (similar to what `bonding` does above)